Source/LightboxController.swift

```diff
@@ -208,9 +208,9 @@ open class LightboxController: UIViewController {
 
     headerView.frame = CGRect(
       x: 0,
-      y: 16,
+      y: 15,
```

Comment:
I bumped this down to 15 basically just to make for more round numbers. Now the header uses up 60px and we make sure the images leave 120px padding by default (so when centered there's 60px for the header).